src/utils/ethereumUtils.js

```diff
@@ -174,7 +174,7 @@ const getBalanceAmount = (selectedGasPrice, selected) => {
     getAsset(assets, selected?.address)?.balance?.amount ??
     0;
 
-  if (selected?.address === ETH_ADDRESS) {
+  if (selected?.isNativeAsset) {
```

Comment:
this property does not exist, we should add when we parse all assets or use the util in `handlers/assets.js`